tests/python_package_test/test_sequence.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
```

Comment:
@jameslamb Need your opinion here.

Do we need a separate test file for testing Sequence input? Or this test should go in some of the currently existing test files?

Reply:
Thanks for the `@`.

No, this test should be put near other `Sequence` tests.

After this one, please:

https://github.com/microsoft/LightGBM/blob/830763d9e58e05b678fb88e9b6f574a20158a36a/tests/python_package_test/test_basic.py#L203